components/PlantList.js

```diff
@@ -0,0 +1,32 @@
+/* Was macht diese Seite?
+- gibt eine Liste von allen Elementen in assets/plants.js wieder
+- map + rendered alles in einer PlantCard wieder 
+*/
+
+import { plants } from "@/assets/plants";
+import styled from "styled-components";
+import PlantCard from "./PlantCard";
+
+export default function PlantList() {
+  return (
+    <div>
+      <StyledHeader>My Plant Collection</StyledHeader>
+      <StyledList>
+        {plants.map((plant) => (
+          <li key={plant.id}>
+            <PlantCard plant={plant} />
+          </li>
+        ))}
+      </StyledList>
```

Comment:
What if there are no plants? If that is a scenario that could happen, then we would end up with an empty `<ul>`.

If the plants could be empty at some point, then we would probably want to do a check before displaying a `<ul>`.